vault/resource_terraform_cloud_secret_backend_test.go

```diff
@@ -3,27 +3,26 @@ package vault
 import (
 	"fmt"
 	"os"
-	"strings"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 
-	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
 	"github.com/hashicorp/terraform-provider-vault/testutil"
 )
 
 func TestTerraformCloudSecretBackend(t *testing.T) {
 	backend := acctest.RandomWithPrefix("tf-test-terraform-cloud")
 	token := os.Getenv("TEST_TF_TOKEN")
 
-	resourceName := "vault_terraform_cloud_secret_backend.test"
+	resourceType := "vault_terraform_cloud_secret_backend"
+	resourceName := resourceType + ".test"
 
 	resource.Test(t, resource.TestCase{
 		Providers:    testProviders,
 		PreCheck:     func() { testutil.TestAccPreCheck(t) },
-		CheckDestroy: testAccTerraformCloudSecretBackendCheckDestroy,
+		CheckDestroy: testCheckMountDestroyed(resourceType, consts.MountTypeGCP, consts.FieldBackend),
```

Comment:
I see that `resource_okta_auth_backend_test.go` also is using a custom Mount Check Destroy function. Should we also update that here?

Reply:
fixing....

Reply:
Fixed with: dd72c7d9e92a130346a8b4fd38407e6e7ce3897d